From 05959f0fe4cdef9b79eb3a33592d76608c3f5ee0 Mon Sep 17 00:00:00 2001
Message-Id: <patch-4.7-05959f0fe4c-20220310T134811Z-avarab@gmail.com>
In-Reply-To: <cover-0.7-00000000000-20220310T134811Z-avarab@gmail.com>
References: <cover.1646390152.git.dyroneteng@gmail.com>
	<cover-0.7-00000000000-20220310T134811Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 4 Mar 2022 12:32:07 +0100
Subject: [RFC/REVIEW 4/7] ls-tree: remove unused "MODE_UNSPECIFIED"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove the "MODE_UNSPECIFIED" added in a preceding commit. We don't
need it, instead let's start the "command mode" enum at 1, as
e.g. builtin/help.c does for its similar pattern.

In addition let's change this from "mutx_option" to a more obvious
"ls-tree_cmdmode", even if the type name isn't used anywhere else. The
previous name sounded as though it had something to do with
a (thread?) mutex.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/ls-tree.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/builtin/ls-tree.c b/builtin/ls-tree.c
index f04733a9be1..95816709d38 100644
--- a/builtin/ls-tree.c
+++ b/builtin/ls-tree.c
@@ -49,11 +49,10 @@ static const  char * const ls_tree_usage[] = {
 	NULL
 };
 
-static enum mutx_option {
-	MODE_UNSPECIFIED = 0,
+static enum ls_tree_cmdmode {
+	MODE_LONG = 1,
 	MODE_NAME_ONLY,
 	MODE_OBJECT_ONLY,
-	MODE_LONG,
 } cmdmode;
 
 static void expand_objectsize(struct strbuf *line, const struct object_id *oid,
-- 
2.35.1.1295.g6b025d3e231

